components/post_view/post_info/index.js

```diff
@@ -4,18 +4,24 @@
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 import {addReaction, removePost} from 'mattermost-redux/actions/posts';
+import {makeGetChannel} from 'mattermost-redux/selectors/entities/channels';
 import {get, getBool} from 'mattermost-redux/selectors/entities/preferences';
 
 import {Preferences} from 'utils/constants.jsx';
 
 import PostInfo from './post_info.jsx';
 
-function mapStateToProps(state, ownProps) {
-    return {
-        ...ownProps,
-        useMilitaryTime: getBool(state, Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.USE_MILITARY_TIME, false),
-        isFlagged: get(state, Preferences.CATEGORY_FLAGGED_POST, ownProps.post.id, null) != null,
-        isMobile: state.views.channel.mobileView
+function makeMapStateToProps() {
+    const getChannel = makeGetChannel();
+
+    return (state, ownProps) => {
+        return {
+            ...ownProps,
+            teamId: getChannel(state, {id: ownProps.post.channel_id}).team_id,
```

Comment:
I'd suggest to get the channel first and check it's presence before getting `team_id`. Thoughts?

Reply:
Yes, you are right

Reply:
Fixed